### PR TITLE
feat: ability to download source as .xml file

### LINKED ISSUE #399: feature: ability to download source as .xml file
Should just add a little download icon at the top right of the source header that produces an xml mimetype document for download

### PR DESCRIPTION
**Pull request by emaan-c**
This PR Closes #399 

Added a download icon in the source header to allow users to download the source as a .xml file. Also moved the 'copy XML source to clipboard' button in the source header!

image
image
image
image

### CODE REVIEW

File: app/renderer/components/Inspector/Inspector.js [hunk 1]
```diff
@@ -38,6 +39,19 @@ const MIN_WIDTH = 1080;
 const MIN_HEIGHT = 570;
 const MAX_SCREENSHOT_WIDTH = 500;
 
+function downloadXML (sourceXML) {
+  let element = document.createElement('a');
+  element.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(sourceXML));
```
Comment (mykola-mokhnach) [inline app/renderer/components/Inspector/Inspector.js:44]: I believe there is a dedicated mime type for xml

Comment (emaan-c) [reply app/renderer/components/Inspector/Inspector.js:44]: Thanks for the heads up! Making those changes right now.

File: app/renderer/components/Inspector/SelectedElement.js [hunk 2]
```diff
@@ -17,8 +17,20 @@ import { ROW, ALERT } from '../../../../gui-common/components/AntdTypes';
 
 const ButtonGroup = Button.Group;
 const NATIVE_APP = 'NATIVE_APP';
-const selectedElementTableCell = (text) => (
-  <div className={styles['selected-element-table-cells']}>{text}</div>);
+
+function selectedElementTableCell (text, copyToClipBoard) {
+  if (copyToClipBoard) {
+    return <div className={styles['selected-element-table-cells']}>
+      <Tooltip title='Copied!' trigger="click">
```
Comment (mykola-mokhnach) [inline app/renderer/components/Inspector/SelectedElement.js:24]: Shall we localize the tooltip text?

File: app/renderer/components/Inspector/Inspector.js [hunk 3]
```diff
@@ -141,7 +155,17 @@ export default class Inspector extends Component {
...
               <div className='action-col'>
-                <Card title={<span><FileTextOutlined /> {t('App Source')}</span>}>
+                <Card title={<span><FileTextOutlined /> {t('App Source')} </span>}
+                  extra={
+                    <span>
+                      <Tooltip title={t('Copy XML Source to Clipboard')}>
+                        <Button type='text' id='btnSourceXML' icon={<CopyOutlined/>} onClick={() => clipboard.writeText(sourceXML)} />
+                      </Tooltip>
+                      <Tooltip title={t('Download Source as .XML File')}>
+                        <Button type='text' id='btnDownloadSourceXML' icon={<DownloadOutlined/>} onClick={() => downloadXML(sourceXML)}/>
```
Comment (mykola-mokhnach) [inline app/renderer/components/Inspector/Inspector.js:165]: I assume the new string must be added to resources

**Pull request closed without merging**